src/api/roles/coach/coach.py

```
@@ -65,9 +69,188 @@ def create_coach_request(coach_details: CoachRequestInput, db = Depends(get_sess
+        raise HTTPException(404, detail="Coach not found")
+    
+    if coach.verified == False:
+        raise HTTPException(404, detail="Coach is not verified yet, availability is not viewable")
+    
+    availabilities = db.query(Availability).filter(Availability.coach_availability_id == coach.coach_availability).all()
+
+    return CoachAvailabilityResponse(coach_availabilities=availabilities)
+
+
+@router.post("/accept_client_request", response_model=AcceptedClientResponse)
+def accept_client_request(request_input : ClientCoachRequestInput, db = Depends(get_session), acc: Account = Depends(get_client_account)):
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)